src/site/_utils/get-srcset-range.js

```diff
@@ -21,13 +21,14 @@ const srcsetRange = [240, 480, 768, 1045, 1434, 1730, 1959, 2195, 2880, 3200];
 /**
  * Take min and max values for an image and return all srcset sizes that fall
  * within that range.
- * @param {?number} min A minimum width.
- * @param {?number} max A maximum width.
+ * @param {number} min A minimum width.
+ * @param {number} max A maximum width.
  * @return {Array<number>} An array of srcset widths.
  */
-module.exports = function getSrcsetRange(min, max) {
-  min = min || srcsetRange[0];
-  max = max || srcsetRange[srcsetRange.length - 1];
+module.exports = function getSrcsetRange(
+  min = srcsetRange[0],
+  max = srcsetRange[srcsetRange.length - 1],
```

Comment:
It seems like this changes the way the function works so you can no longer pass in your own `min` and `max`

Reply:
You can still pass in your own min and max, but if you don't it get assigned to the srcset values.